### PR TITLE
FINERACT-1724: Add indexes to assets and jobs

### PR DESCRIPTION
**Pull request by b0c1**
FINERACT-1724: Add indexes to assets and jobs
- [x] Add `loan_id` as indexed field on the `m_external_asset_owner_transfer`
- [x] Add `job_execution_id` as indexed field on the `batch_job_execution_params` table

### CODE REVIEW

File: fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml [hunk 1]
```diff
@@ -141,4 +141,5 @@
     <include file="parts/0119_add_loan_product_down_payment_configuration.xml" relativeToChangelogFile="true" />
     <include file="parts/0120_transaction_summary_with_asset_owner_report_typo_fix.xml" relativeToChangelogFile="true" />
     <include file="parts/0121_transaction_summary_with_asset_owner_report_typo_fix_2.xml" relativeToChangelogFile="true" />
+    <include file="parts/0122_add_batch_job_execution_params_index.xml" relativeToChangelogFile="true" />
```
Comment (adamsaghy) [inline fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml:144]: Please move this into the fineract-investor module

**Review by adamsaghy — CHANGES_REQUESTED** [review]
Please see my comments!

**Review by ruchiD — COMMENTED** [review]
MySQL builds are failing due to liquibase exception while creating index.
liquibase.exception.DatabaseException: Table 'fineract_default.batch_job_execution_params' doesn't exist. Please check

**Review by adamsaghy — APPROVED** [review]
LGTM

**Review by reluxa — APPROVED** [review]

**Pull request merged**